support-frontend/assets/pages/contributions-landing/contributionsLandingActions.ts

```diff
@@ -275,23 +275,8 @@ function getBillingCountryAndState(
 	};
 }
 
-function getProductOptionsForBenefitsTest(
-	amount: number,
-	state: ContributionsState,
-) {
-	const isInNewProductTest =
-		state.common.abParticipations.newProduct === 'variant';
-	const contributionType = getContributionType(state);
-	const isRecurring = contributionType !== 'ONE_OFF';
-
-	const thresholdPrice = getThresholdPrice(
-		state.common.internationalisation.countryGroupId,
-		contributionType,
-	);
-	const amountIsHighEnough = !!(thresholdPrice && amount >= thresholdPrice);
-	const shouldGetSupporterPlus =
-		isInNewProductTest && isRecurring && amountIsHighEnough;
-	return shouldGetSupporterPlus
+function getProductOptionsForSupporterPlusTest(state: ContributionsState) {
+	return isSupporterPlusPurchase(state)
```

Comment:
I replaced this logic with a call to `isSupporterPlusPurchase` as this function does the same and returns the correct value.